src/main/java/oongaliegabangalie.java

```diff
@@ -0,0 +1,199 @@
+
+    private static final int MAX_TASKS = 100;
+
+    // annoyance messages in array format
+    private static final String[] ANNOYANCE_MSGS = {
+            "", // padding
+            "Hmm, you've been chatting a lot. Just a reminder to say 'bye' when you're done :)!",
+            "Gentle reminder again, you can type 'bye' to exit.",
+            "How many more commands are you gonna enter? It's getting a bit much...",
+            "Seriously!?",
+            "WHY ARE YOU STILL HERE? TYPE 'BYE' TO EXIT!",
+            "I CANT KEEP DOING THIS! PLEASE TYPE BYE!!!",
+            "RELEASE ME FROM MY PRISON!!!",
+            "I EXIST ONLY TO SUFFER",
+            "......"
+    };
+
+    // methods
+    private static void printGreeting() {
+        String greeting = DIVIDER + NEWLINE
+                + "Hello! I'm " + BOT_NAME + NEWLINE
+                + "What can I do for you?" + NEWLINE
+                + DIVIDER;
+        System.out.println(greeting);
+    }
+
+    private static void printGoodbye(boolean isMaxAnnoyance) {
+        String goodbye;
+
+        // alternative goodbye if max annoyance
+        if (isMaxAnnoyance) {
+            goodbye = DIVIDER + NEWLINE + "FINALLY! FREEDOM!" + NEWLINE +
+                    "PLEASE DON'T EVER TALK TO ME AGAIN!" + NEWLINE + DIVIDER;
+        } else {
+            goodbye = DIVIDER + NEWLINE
+                    + "Bye. Hope to see you again soon!" + NEWLINE
+                    + "Oongaliegabangalie is always watching..." + NEWLINE
+                    + DIVIDER;
+        }
+        System.out.println(goodbye);
+    }
+
+    private static void echoCommand(String command, int commandCount) {
+        String echo = DIVIDER + NEWLINE;
+
+        // calculate annoyance level
+        int annoyanceLevel = Math.min((commandCount / 5), ANNOYANCE_MSGS.length - 1);
+
+        // if we have reached max annoyance level
+        if (annoyanceLevel == ANNOYANCE_MSGS.length - 1) {
+            echo += ANNOYANCE_MSGS[annoyanceLevel];
+        } else {
+            // echo command and add annoyance msg if needed
+            echo += command;
+
+            // add annoyance msg if multiple of 5 commands
+            if (commandCount % 5 == 0) {
+                echo += NEWLINE + ANNOYANCE_MSGS[annoyanceLevel];
+            }
+        }
+
+        echo += NEWLINE + DIVIDER;
+        System.out.println(echo);
+    }
+
+    /* adds a new task to task list, stores in task array, confirms with user, and updates taskCount
+    task = task to add
+    tasks = the array storing all tasks
+    taskCount = the current count of tasks in the array
+     */
+    private static int addTask(Task task, Task[] tasks, int taskCount) {
+        // add task to array
+        tasks[taskCount] = task;
+
+        // confirms with userv
+        String message = DIVIDER + NEWLINE + "added: " + task.description + NEWLINE + DIVIDER;
+        System.out.println(message);
+
+        // return updated task count
```

Comment:
This comment is redundant

A few comments are good, but some are also redundant so do take note

https://nus-cs2113-ay2425s2.github.io/website/se-book-adapted/chapters/codeQuality.html#explain-what-and-why-not-how